runners/direct-java/src/test/java/org/apache/beam/runners/direct/EvaluationContextTest.java

```diff
@@ -101,8 +101,8 @@ public void setup() {
     view = created.apply(View.<Integer>asIterable());
     unbounded = p.apply(GenerateSequence.from(0));
 
-    KeyedPValueTrackingVisitor keyedPValueTrackingVisitor = KeyedPValueTrackingVisitor.create();
-    p.traverseTopologically(keyedPValueTrackingVisitor);
+    DirectGraphVisitor graphVisitor = new DirectGraphVisitor();
+    p.traverseTopologically(graphVisitor);
 
     BundleFactory bundleFactory = ImmutableListBundleFactory.create();
     graph = DirectGraphs.getGraph(p);
```

Comment:
We can either use the existing `DirectGraphVisitor` or add a utility method to `DirectGraphs`. I'm in favor of the former.